authoring-new-actions.md

```diff
@@ -0,0 +1,90 @@
+
+To help GH developers easily create GitHub CI/CD workflows targeting Azure, we also published [Azure starter templates](https://github.com/Azure/actions-workflow-samples) to deploy apps created with popular languages and frameworks such as .NET, Node.js, Java, PHP, Ruby or Python, in containers or running on any operating system.
+
+GitHub Actions for Azure are also integrated into Visual Studio Code, Azure CLI, and the Azure Portal simplifying the experience of [deploying to Azure from your favorite tools.](https://azure.microsoft.com/en-us/blog/deploy-to-azure-using-github-actions-from-your-favorite-tools/)
+
+This document is intended to help developers with the recommended process to create, publish and contribute to actions for Azure. Please reach out to the [vTeam](mailto:) for any queries.
+
+# Creating a new GitHub Action for Azure or Microsoft
+
+You can [create your own actions](https://docs.github.com/en/free-pro-team@latest/actions/creating-actions), use and customize actions shared by the GitHub community, or write and share the actions you build.
+
+This guidance is to enable developers and teams intending to create a new GitHub Action targeting Azure. 
+
+## Before you author
+
+- Kindly confirm that the new action would enable a new developer scenario that can not be accomplished by the existing set of [Actions for Azure](https://github.com/marketplace?type=actions&query=Azure) or by enhancing them through contributions.
+- Choose the right GitHub org to create a repo. For example, for all Actions targeting an Azure service, choose [https://github.com/Azure/](https://github.com/Azure/) as the Org. Else for any generic utility actions which are cloud agnostic, chose [https://github.com/Microsoft](https://github.com/Microsoft) as the org for hosting the Action repo.
+- Ensure that only one action is present per repo. Create new repo for each of your actions.
+
+## Steps to create a new Action repo 
+
+Request a new repo from the below links as suitable:
+  - [https://repos.opensource.microsoft.com/orgs/Azure/new](https://repos.opensource.microsoft.com/Azure) for creating repos under github.com/Azure Org
+  - [https://repos.opensource.microsoft.com/orgs/Microsoft/new](https://repos.opensource.microsoft.com/microsoft) for creating repos under github.com/Microsoft Org
+
+Ensure to select **Engineering system: GitHub for Open Source** as GitHub Actions for Azure or Microsoft are published as Open Source in repos created
+Microsoft's official GitHub.com organizations, powered by GitHub Enterprise Cloud.
```

Comment:
nit: in repos created **in/under** Microsoft's official..?

Reply:
corrected.